server/models/approvedActionPlanSummary.ts

```diff
@@ -1,4 +1,5 @@
 export default interface ApprovedActionPlanSummary {
   id: string
   approvedAt: string
+  submittedAt: string
```

Comment:
does this need a contract test change too?

Reply:
the contract tests use the factory function which is also changed in this PR. 

it's not super obvious, but this is now encoded in the pacts.

Reply:
ah cool -- thanks